### ISSUE TITLE
Can't play mp4 online

### ISSUE THREAD

**Issue by ZTMIDGO** [swhitty/FlyingFox#120]
let video = PhotosPickerItem.loadTransferable(type: Data.self) to get the video data and pass it to server.appendRoute return HTTPResponse(statusCode:.ok, body: video!),Can't play mp4 online

**Comment by swhitty** [comment]
Hi, does the file download correctly using a downloader like `wget` or `curl` ?

Very large files should download correctly and most browsers will simply download the mp4 and play it.

**Comment by ZTMIDGO** [comment]
Yes, server.appendRoute downloads files correctly, can FlayFox do static resource management similar to Jetty, I can use Jetty for online playback, I don't know if FlayFox needs to add headers to HTTPResponse for online playback
